src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface PaginationProps {
+  arrItems: string[];
+  total: number;
+  perPage: number;
+  setCurrentPage: (page: number) => void;
+  currentPage: number;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  arrItems,
+  total,
+  perPage,
+  setCurrentPage,
+  currentPage,
+}) => {
+  const pageNumbers: number[] = [];
+
+  for (let i = 1; i <= Math.ceil(total / perPage); i++) {
+    pageNumbers.push(i);
+  }
+
+  const totalPages = getNumbers(1, Math.ceil(total / perPage)).length;
```

Comment:
The totalPages calculation is redundant since it's essentially the same as the length of the pageNumbers array you just created. You can use the pageNumbers.length to get the total number of pages.